### PR TITLE
Create a workflow to suggest lintrunner fixes

### PR DESCRIPTION
**Pull request by justinchuby**
Suggest fixes automatically so contributors can apply formatting changes without needing to do so locally

### CODE REVIEW

File: onnx/external_data_helper.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 from onnx.onnx_pb import (
     AttributeProto,
     FunctionProto,
+
```
Comment (liqunfu) [inline onnx/external_data_helper.py:17]: extra empty lines?

File: onnx/external_data_helper.py [hunk 2]
```diff
@@ -29,7 +31,9 @@ def __init__(self, tensor: TensorProto) -> None:
         self.basepath = ""
 
         for entry in tensor.external_data:
-            setattr(self, entry.key, entry.value)
+            setattr(self, entry.key,
+
```
Comment (liqunfu) [inline onnx/external_data_helper.py:35]: same extra empty lines

File: onnx/external_data_helper.py [hunk 3]
```diff
@@ -48,7 +52,7 @@ def load_external_data_for_tensor(tensor: TensorProto, base_dir: str) -> None:
     """
     info = ExternalDataInfo(tensor)
     external_data_file_path = c_checker._resolve_external_data_location(  # type: ignore[attr-defined]
-        base_dir, info.location, tensor.name
+        base_dir, info.location, tensor.name
```
Comment (liqunfu) [inline onnx/external_data_helper.py:55]: additional space?

**Review by liqunfu — APPROVED** [review]

File: onnx/external_data_helper.py (hunk 3 above)
Comment (justinchuby) [reply onnx/external_data_helper.py:55]: I am using these to test the workflow

**Comment by justinchuby** [comment]
The workflow is not working very well yet, due to https://github.com/parkerbxyz/suggest-changes/issues/36 and https://github.com/parkerbxyz/suggest-changes/issues/36

**Comment by justinchuby** [comment]
https://github.com/parkerbxyz/suggest-changes/issues/37 is fixes, so we can merge.

**Pull request merged**